soc/wch/ch32v/qingke_v4f/vector.S

```diff
@@ -22,11 +22,17 @@ SECTION_FUNC(vectors, ivt)
 	lui	x5, 0x8000
 	jr	0x8(x5)
 	j       __start
+_irq_vector_table:
 	.rept   CONFIG_VECTOR_TABLE_SIZE
 	.word	_isr_wrapper
 	.endr
 
 SECTION_FUNC(vectors, __start)
-	li 	a0, 0xf
+	li      a0, 0x1f
+	csrw    0xbc0, a0
+	li      a0, 0x1e
+	csrw    0x804, a0
+	la t0, _irq_vector_table /* Load address of interrupt vector table */
+	ori t0, t0, 0x03
 	csrw	mtvec, a0
```

Comment:
Just to double check: on the v2a and v2c, mtvec must be aligned to a 1 KiB boundrary else it will halt. Does the v4f support looser alignments?

Reply:
I am actually going to revert this particular bit. I plan to work on support for flashing with different offsets to support using tinyuf2, but don't need to wrap that work into this PR.